yarn.lock

```diff
@@ -14301,97 +14302,195 @@ __metadata:
   languageName: node
   linkType: hard
 
+"esbuild-android-64@npm:0.14.25":
+  version: 0.14.25
+  resolution: "esbuild-android-64@npm:0.14.25"
+  conditions: os=android & cpu=x64
+  languageName: node
+  linkType: hard
+
```

Comment:
Code looks fine to me, I'm just a little nervous that in the lock file here's duplicating all these esbuild dependencies with a slightly different version... 0.14.27 -> 0.14.25. In the package.json, if you change the version to 0.14.27 do all these duplicates go away?

Reply:
oh, let me check that. I needed to install that lib but it asked me if I wanted to use the version it was currently being used or latest and I chose existing.

Reply:
I checked and the renovate[bot] updated the version number 13 hs ago so this ended up being outdated. I already updated the version in redwood record manually.